src/main/java/org/opensearch/jobscheduler/JobSchedulerPlugin.java

```diff
@@ -60,10 +79,12 @@ public class JobSchedulerPlugin extends Plugin implements ExtensiblePlugin {
     private LockService lockService;
     private Map<String, ScheduledJobProvider> indexToJobProviders;
     private Set<String> indicesToListen;
+    private HashMap<String, JobDetails> jobDetailsHashMap;
```

Comment:
Just declare this as a `Map` and remove `hash` from the name.

Reply:
ACK